crates/types/src/joints.rs

```diff
@@ -720,6 +720,60 @@ pub struct JointsCommand<T> {
     pub positions: Joints<T>,
     pub stiffnesses: Joints<T>,
 }
+impl_Interpolate!(f32, JointsCommand<f32>, PI);
+
+impl JointsCommand<f32> {
+    pub fn mirrored(self) -> Self {
+        Self {
+            positions: Joints::mirrored(self.positions),
+            stiffnesses: Joints::mirrored(self.stiffnesses),
+        }
+    }
+}
+
+impl Mul<f32> for JointsCommand<f32> {
+    type Output = JointsCommand<f32>;
+
+    fn mul(self, right: f32) -> Self::Output {
+        Self::Output {
+            positions: Joints::mul(self.positions, right),
+            stiffnesses: self.stiffnesses * right,
+        }
+    }
+}
```

Comment:
I'm not sure of this implementation.
While it makes sense in the context of interpolating between keyframes, it could be confusing when used in other places.
Without knowing the what this implementation does exactly, I might expect that multiplying a joints command by 0.5 would only change the joint angles.

Reply:
I tried a different approach, where I created a new type `JointsWithStiffnesses` to implement the interpolation. To avoid to not interpolate `JointsCommand` at an other position. Maybe you could have a look. 
I didn't looked over the motion again, because I think the motion can be changed in a different PR.